src/features/rewards/boxAlert/style.ts

```diff
@@ -25,11 +27,11 @@ export const StyledAlertIcon = styled<{}, 'div'>('div')`
   color: #15A4FA;
 `
 
-export const StyledInfo = styled<{}, 'div'>('div')`
+export const StyledInfo = styled<StyleProps, 'div'>('div')`
   font-size: 14px;
   letter-spacing: 0;
   line-height: 18px;
-  padding: 12px 15px 0px 11px;
+  padding: ${p => p.type === 'tips' ? '12' : '20'}px 15px 0px 11px;
```

Comment:
let's use flex for this so that is aligned to the middle of the icon and we don't need to adjust padding for every type

Reply:
done